### PR TITLE
Tickets/dm 38790 - add memo

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 1]
```diff
@@ -398,6 +401,27 @@
...
 \def\@controlFootText{}
 
+\newcommand{\mkmemotitle}{
+        \pagestyle{empty}
+        \vspace*{-30pt}
+
+        \includegraphics[width=2.5cm]{rubin_logo}
+        \hfill
+        \begin{minipage}{0.2\textwidth}
+        \large
```
Comment (leannep) [inline texmf/tex/latex/lsst/lsstdoc.cls:412]:
```suggestion
        \begin{flushright}
        \vspace*{-50pt}
        \large 
```

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 2]
```diff
@@ -398,6 +401,27 @@
 \def\@controlText{}
 \def\@controlFootText{}
 
+\newcommand{\mkmemotitle}{
+        \pagestyle{empty}
+        \vspace*{-30pt}
+
+        \includegraphics[width=2.5cm]{rubin_logo}
+        \hfill
+        \begin{minipage}{0.2\textwidth}
```
Comment (leannep) [inline texmf/tex/latex/lsst/lsstdoc.cls]:
```suggestion
        \begin{minipage}{0.3\textwidth}
```

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 3]
```diff
@@ -398,6 +401,27 @@
 \def\@controlText{}
 \def\@controlFootText{}
 
+\newcommand{\mkmemotitle}{
+        \pagestyle{empty}
+        \vspace*{-30pt}
+
+        \includegraphics[width=2.5cm]{rubin_logo}
```
Comment (leannep) [inline texmf/tex/latex/lsst/lsstdoc.cls]:
```suggestion
        \includegraphics[width=4.0cm]{rubin_logo}
```

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 4]
```diff
@@ -398,6 +401,27 @@
...
+        \hfill
+        \begin{minipage}{0.2\textwidth}
+        \large 
+        \color{lssttitle}
```
Comment (leannep) [inline texmf/tex/latex/lsst/lsstdoc.cls:413]:
```suggestion
        \color{black}
```

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 5]
```diff
@@ -398,6 +401,27 @@
...
+        950 Cherry Ave.,
```
Comment (leannep) [inline texmf/tex/latex/lsst/lsstdoc.cls:414]:
```suggestion
        950 North Cherry Avenue \\
```

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 6]
```diff
@@ -398,6 +401,27 @@
...
+        85719 Tucson.
```
Comment (leannep) [inline texmf/tex/latex/lsst/lsstdoc.cls:415]:
```suggestion
        Tucson, AZ, 85719 \\
```

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 7]
```diff
@@ -398,6 +401,27 @@
...
+        \color{black}
+	\docDate
```
Comment (leannep) [inline texmf/tex/latex/lsst/lsstdoc.cls:417]:
```suggestion
	\vspace{3mm}
	\textbf{\docDate} 
	\end{flushright}
```

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 8]
```diff
@@ -398,6 +401,27 @@
...
+        950 Cherry Ave.,
+        85719 Tucson.
+        \color{black}
+	\docDate 
+        \end{minipage}
+      
+       \vspace{10pt}
+       \textbf {To: \memoto}\\
+       \textbf {From: \@author}\\
+        \textbf {Subject: \@title }\\
```
Comment (leannep) [inline texmf/tex/latex/lsst/lsstdoc.cls]:
```suggestion

        \textbf {Subject: \@title }\\
```

**Review by leannep — CHANGES_REQUESTED** [review]
Thank you for this - I have taken it for a test run and made a few proposed change

File: texmf/tex/latex/lsst/lsstdoc.cls (hunk 4 above)
Comment (womullan) [reply texmf/tex/latex/lsst/lsstdoc.cls:415]: i liked the teal, but ok

**Comment by womullan** [comment]
changes incorporated

**Pull request merged**